Rakefile

```diff
@@ -20,8 +20,6 @@ load 'lib/tasks/tax_cloud.rake'
 load 'lib/tasks/tax_codes.rake'
 load 'lib/tasks/tax_code_groups.rake'
 
-load 'vcr/tasks/vcr.rake'
```

Comment:
Had an exception during load the file about its missing. Not sure that the change is correct.